cmw.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmw
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/fxamacker/cbor/v2"
+)
+
+type Serialization uint
+
+const (
```

Comment:
if I am not mistaken, this is more than the README stated specification.

This could lead to confusion for new readers.

Please clarify it in some way in README.md

Reply:
this is tracking ahead :-)